### ISSUE TITLE
why remove "background" class?

### ISSUE THREAD

**Issue by boundles** [chengyangfu/retinamask#14]
"box_cls" shape is num_class-1, so why remove "background" class?

**Comment by chengyangfu** [comment]
This follows the implementation of origianl FocalLoss paper. FocalLoss is a weighted version of multi-class sigmoid cross entropy loss function.  Each class is treated independently. I also tried the softmax version but didn't get better results.

**Comment by boundles** [comment]
OK, got it, thanks for your reply.
